README.md

```diff
@@ -0,0 +1,10 @@
+### Cafe
+
+The Beemo Cafe is a mono-repo containing all the public microservices that Beemo will, or is utilizing. In this mono-repo contains the following services:
```

Comment:
I'd just say that the stuff here actually powers Beemo, not that it may or may not be currently powering it now or in the future, that sounded a bit confusing. Also, a bit of grammar at the end 😄

```suggestion
The Beemo Cafe is a monorepo containing all the public microservices that are powering Beemo. The repo consists of the following services:

```